examples/chart/teleport-cluster/templates/auth/_config.azure.tpl

```diff
@@ -0,0 +1,13 @@
+{{- define "teleport-cluster.auth.config.azure" -}}
+{{ include "teleport-cluster.auth.config.common" . }}
+  storage:
+    type: postgresql
+    auth_mode: azure
+    conn_string: "host={{.Values.azure.databaseHost}}.postgres.database.azure.com sslmode=verify-full user={{.Values.azure.databaseUser}} dbname={{.Values.azure.backendDatabase}}{{with .Values.azure.databasePoolMaxConnections}} pool_max_conns={{ . }}{{- end}}"
+    audit_sessions_uri: azblob://{{.Values.azure.sessionRecordingStorageAccount}}.blob.core.windows.net
+    audit_events_uri:
+      - "postgresql://{{.Values.azure.databaseUser}}@{{.Values.azure.databaseHost}}.postgres.database.azure.com/{{.Values.azure.auditLogDatabase}}?sslmode=verify-full#auth_mode=azure"
```

Comment:
I might be overcomplicating it and it doesn't exclude the pool_max_conns value if it's zero, but
```suggestion
    conn_string: {{ urlJoin (dict
      "scheme" "postgresql"
      "userinfo" .Values.azure.databaseUser
      "host" (printf "%v.postgres.database.azure.com" .Values.azure.databaseHost)
      "path" .Values.azure.backendDatabase
      "query" (printf "sslmode=verify-full&pool_max_conns=%v" (urlquery .Values.azure.databasePoolMaxConnections))
    ) | toYaml }}
    audit_sessions_uri: {{ urlJoin (dict
      "scheme" "azblob"
      "host" (printf "%v.blob.core.windows.net" .Values.azure.sessionRecordingStorageAccount)
    ) | toYaml }}
    audit_events_uri:
      - {{ urlJoin (dict
          "scheme" "postgresql"
          "userinfo" .Values.azure.databaseUser
          "host" (printf "%v.postgres.database.azure.com" .Values.azure.databaseHost)
          "path" .Values.azure.auditLogDatabase
          "query" "sslmode=verify-full"
          "fragment" "auth_mode=azure"
        ) | toYaml }}
```

Reply:
(I don't know how values work, the `urlquery .Values.azure.databasePoolMaxConnections` might be useless if the value is guaranteed to be an integer)